generator/src/main/java/com/scottlogic/deg/generator/constraints/atomic/AtomicConstraintsHelper.java

```diff
@@ -0,0 +1,16 @@
+package com.scottlogic.deg.generator.constraints.atomic;
+
+import com.scottlogic.deg.generator.Field;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class AtomicConstraintsHelper {
```

Comment:
A type already exists that does very similar behaviour to this, see `AtomicConstraintCollector`, it does take a `DecisionTree`, but makes sense to extend this to take a `ConstraintNode` (as an overload?) 

This type/method is always called given the output of `ConstraintNode.getAtomicConstraints()`

Reply:
That checks for all constraints in a tree, rather than in a node